Tests/RxSwiftTests/Observable+BufferTests.swift

```diff
@@ -142,4 +142,68 @@ extension ObservableBufferTest {
             scheduler.start()
         }
     #endif
+    
+    func testBufferWithTrigger_Default() {
+        let testScheduler = TestScheduler(initialClock: 0)
+        
+        let hot = testScheduler.createHotObservable([
+            .next(205, 1),
+            .next(210, 2),
+            .next(240, 3),
+            .next(255, 4),
+            .next(320, 5),
+            .next(350, 6),
+            .next(370, 7),
+            .next(875, 8),
+            .next(900, 9),
+            .next(999, 10),
+            .completed(1000)
+            ]).share()
+        
+        let debounced = hot.debounce(.seconds(30), scheduler: testScheduler)
```

Comment:
What's the reason for the debouncing here? might be missing something

Reply:
I don't remember so I am not sure haha. I'll have to test whether that has any affect or why it may be there.

Reply:
Oh, well I guess this was the quick and dirty "generic" trigger I came up with in the heat of the moment. It is essentially the same thing as `buffer(debounce:)` though so one could change this one for something else. Any ideas? Maybe something that counts? Makes sure each buffer is of a specific size, perhaps?

Reply:
Manual signal sounds good, thanks!